jekyll.gemspec

```diff
@@ -38,6 +38,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency("liquid",                "~> 4.0")
   s.add_runtime_dependency("mercenary",             "~> 0.3.3")
   s.add_runtime_dependency("pathutil",              "~> 0.9")
-  s.add_runtime_dependency("rouge",                 "~> #{ENV["ROUGE_VERSION"] || "1.7"}")
+  s.add_runtime_dependency("rouge",                 "~> #{ENV["ROUGE_VERSION"] || "2.1.0"}")
```

Comment:
Does this work with 2.0 & 2.2, or is only 2.1.x compatible? Usually we'd just use 2 digits here, e.g. 2.0.